docs/references/react/use-user.mdx

```diff
@@ -103,6 +103,61 @@ export default function Home() {
 ```
 </CodeBlockTabs>
 
+### Reload user data
+
+If you need to retrieve the latest user data after updating the user elsewhere, use the `user.reload()` method.
+
+<CodeBlockTabs options={['Next.js', 'React']}>
```

Comment:
Should this be in <Tabs> and then for the Next.js tab, have a <CodeBlockTabs> for pages vs app router?

Reply:
So the snippet wouldn't actually change in either case, just the filename. I'm not positive it's worth it to create the extra layers there if the snippet is unchanged. What do you think?

Reply:
oh cool ! agreed ^~^